src/Sulu/Bundle/AdminBundle/Resources/js/components/Select/Divider.js

```diff
@@ -0,0 +1,9 @@
+// @flow
+import React from 'react';
+import dividerStyles from './divider.scss';
+
+export default class Divider extends React.PureComponent {
+    render() {
+        return <li className={dividerStyles.divider} />;
```

Comment:
According to the standard (http://w3c.github.io/html/grouping-content.html#the-ul-element) an `ul` is only allowed to have `li`s as children.